### ISSUE TITLE
2. Formularz

### ISSUE THREAD

**Issue by CezaryDanielNowak** [CezaryDanielNowak/10clouds-zestawiator#2]
![formularz](https://i.imgur.com/n2l0Z1J.png)

**Comment by sztyborek** [comment]
@CezaryDanielNowak Czy w polu "Opis dzieł" dane mają być od razu automatycznie zaciągane (np. dla ticketów na które użytkownik poświęcił najwięcej czasu) czy zostawione puste, dopóki sam czegoś ie wybierze? Ja bym się skłaniała ku pierwszemu rozwiązaniu.

**Comment by CezaryDanielNowak** [comment]
@sztyborek też skłaniam się ku Twojemu pomysłowi :) Dodajmy ticket na automatycznie uzupełnianie opisu dzieł na podstawie ticketów.

Jednak jest kilka możliwych scenariuszy uzupełniania jiry:
- jeden ticket zbiorczy z wieloma worklogami (np stepstone tak miał)
- wiele ticketów, każdy z małą liczbą godzin (trzeba by wtedy sprawdzać tytuł parent storiesa)

Problemem moze byc to, ze nazwa ticketu nie jest opisem dzieła... , ale mam też lekkie obawy czy się zmieścimy w czasie

**Comment by sztyborek** [comment]
@CezaryDanielNowak racja, nie pomyślałam o tym, że trzeba też sprawdzać parent stories ticketa itp. Może na początek załóżmy, że robimy podstawową wersję, czyli zostawiamy to pole do wypełnienia użytkownikowi + podpowiedź w tooltipie, a jeśli się zmieścimy w czasie to dodamy autouzupełnianie?
